<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>286</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="HoursMinus">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>H -</string>
    </property>
   </widget>
   <widget class="QPushButton" name="HoursPlus">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>19</pointsize>
     </font>
    </property>
    <property name="text">
     <string>H +</string>
    </property>
   </widget>
   <widget class="QPushButton" name="MinutesMinus">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>90</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>M -</string>
    </property>
   </widget>
   <widget class="QPushButton" name="MinutesPlus">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>20</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>M +</string>
    </property>
   </widget>
   <widget class="QPushButton" name="CoffeeNow">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>170</y>
      <width>161</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Coffee now !</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SetAlarm">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>30</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>19</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Set alarm</string>
    </property>
   </widget>
   <widget class="QLabel" name="WakeUpTime">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>210</y>
      <width>131</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>36</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(0, 0, 0);
background-color: rgb(186, 189, 182);</string>
    </property>
    <property name="text">
     <string>00:00</string>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignHCenter|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QLabel" name="CurrentTime">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>130</y>
      <width>131</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>36</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(0, 0, 0);
background-color: rgb(186, 189, 182);</string>
    </property>
    <property name="text">
     <string>00:00</string>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignHCenter|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>180</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Current time</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>260</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Wake up time</string>
    </property>
   </widget>
   <widget class="QPushButton" name="StopAlarm">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>80</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>19</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Stop alarm</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>-40</x>
      <y>-40</y>
      <width>521</width>
      <height>391</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="textFormat">
     <enum>Qt::MarkdownText</enum>
    </property>
    <property name="pixmap">
     <pixmap>wallpaper-6380532_960_720.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="Quit">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>0</y>
      <width>41</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Quit</string>
    </property>
   </widget>
   <zorder>label_3</zorder>
   <zorder>HoursMinus</zorder>
   <zorder>HoursPlus</zorder>
   <zorder>MinutesMinus</zorder>
   <zorder>MinutesPlus</zorder>
   <zorder>CoffeeNow</zorder>
   <zorder>SetAlarm</zorder>
   <zorder>WakeUpTime</zorder>
   <zorder>CurrentTime</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>StopAlarm</zorder>
   <zorder>Quit</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
